### DISCUSSION TITLE
What English word has three consecutive double letters?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#1015] [Galaxy Trivia] [answered]
**Bonus:** What English word has all five vowels in alphabetical order? 🤓

**Comment by peterbaccaro** [comment]
The English word that has three consecutive double letters is **bookkeeper**. In bookkeeper, the consecutive double letters are **oo**, **kk**, and **ee**.

Some English words that has all five vowels (a, e, i, o, u) in alphabetical order are:

- **Abstemious**: Showing self-discipline, especially with regard to eating or drinking. It refers to someone who exercises moderation or self-restraint, particularly in indulging in pleasures like food and drink.

- **Facetious**: Treating serious issues with inappropriate humour or sarcasm. It describes behaviour or remarks that are intended to be amusing or funny but are often considered flippant or inappropriate in a serious context.

- **Abstemiously**: In a self-disciplined or restrained manner. It denotes the way in which someone demonstrates moderation or self-control, typically related to consumption habits or behaviours.

- **Graciousness**: Kindness, politeness, and charm in one's behaviour. It refers to being courteous, considerate, and showing good manners towards others.

- **Auspiciousness**: The quality of being favourable or conducive to success. It suggests that something has the potential to bring good fortune, positive outcomes, or favourable circumstances.

**Comment by Ben74x** [comment]
"**_Sweet-toothed_**" is the answer for the main question (someone who can't resist sugary treats :laughing:).  I will go with the  lofty "**_aerious_**" for the bonus question. This word's so high up in the clouds, it's often forgotten by us land-dwellers :smile:

**Comment by ParanoidUser** [comment]
**Extra:** What do you call a group of singing 🎤🎶 and 🕺💃 verbs?

**Reply by Ben74x** [reply]
The answer is **Conjugation**

**Reply by peterbaccaro** [reply, marked as answer]
The verb that means to sing and dance simultaneously is to **perform**. When someone performs, they are engaging in a combination of singing and dancing as part of a live performance or presentation.

The verbs "dance" and "sing" are both **action verbs**. They describe physical activities or actions performed by a subject.

**Comment by thuongtruong1009** [comment]
The English word that has three consecutive double letters is "bookkeeper." It has the consecutive double letters "oo," "kk," and "ee."

The English word that has all five vowels (a, e, i, o, u) in alphabetical order is "abstemious."

**Comment by virajchandra51** [comment]
The English word that has three consecutive double letters is "bookkeeper." It has the consecutive double letters "oo," "kk," and "ee."

The English word that has all five vowels (a, e, i, o, u) in alphabetical order is "abstemious."
